clients/client-java/src/main/java/com/datastrato/gravitino/client/BaseSchemaCatalog.java

```diff
@@ -78,7 +75,7 @@ public String[] listSchemas() throws NoSuchCatalogException {
 
     EntityListResponse resp =
         restClient.get(
-            formatSchemaRequestPath(schemaNamespace),
+            formatSchemaRequestPath(Namespace.of(catalogNamespace.level(0), this.name())),
```

Comment:
Maybe we can make this `Namespace.of(catalogNamespace.level(0), this.name())` as a helper method to reduce the duplications.

Reply:
Done, abstract to a method called "schemaNamespace()"